src/components/CommentCard.jsx

```diff
@@ -23,6 +42,15 @@ export default function CommentCard({ comment }) {
             Posted on:{' '}
             <span style={{ fontWeight: 'bold' }}>{readableDate}</span>
           </div>
+          {author === user.username && !confirmDelete ? (
+            <button onClick={() => setConfirmDelete(true)}>Delete</button>
+          ) : null}
+          {confirmDelete ? (
+            <div>
+              Are you sure? <button onClick={handleDelete}>Yes</button> /{' '}
+              <button onClick={() => setConfirmDelete(false)}>No</button>
```

Comment:
As an extra thing, could look at modals to do this. One possible package - https://www.npmjs.com/package/react-modal